There is no (process-kill-without-query) anymore, use
(set-process-query-on-exit-flag) instead.

---
 emacs/uim-helper.el |    2 +-
 emacs/uim.el        |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- emacs/uim-helper.el
+++ emacs/uim-helper.el	2020-08-18 11:46:01.236192703 +0000
@@ -106,7 +106,7 @@
     (if (not proc)
 	(error "uim.el: Couldn't invoke uim-el-helper-agent."))
 
-    (process-kill-without-query proc)
+    (set-process-query-on-exit-flag proc nil)
 
     ;; wait "OK"
     (let ((patience uim-startup-timeout) (ok nil))
--- emacs/uim.el
+++ emacs/uim.el	2020-08-18 11:45:51.828359337 +0000
@@ -488,7 +488,7 @@
 	(error "uim.el: Couldn't invoke uim-el-agent."))
     
     ;; don't ask kill
-    (process-kill-without-query proc)
+    (set-process-query-on-exit-flag proc nil)
 
     ;; wait "OK"
     (let ((patience uim-startup-timeout) (ok nil))
